starcounter-include.html

```diff
@@ -63,26 +64,32 @@
                             });
                         }
                         starcounterInclude.defaultComposition = mergedComposition;
-
                         if (this.model != starcounterInclude.partial) {
                             this.model = starcounterInclude.partial
                         }
                         // notify composition property
                         // TODO: consider moving entire partialChanged or appendChild here
                         starcounterInclude._compositionChanged();
                     });
+                    
                     href && (importedTemplate.href = href);
 
-
                     this.appendChild(importedTemplate);
                     this.template = importedTemplate;
                 }
             }
-            connectedCallback(){
-                this.stampImportedTemplate();
+            // computed method name (http://2ality.com/2015/02/es6-classes-final.html#computed-method-names)
+            [WebComponentsV0 ? 'attachedCallback' : 'connectedCallback']() {
+                this.stampImportedTemplate();   
             }
-        }
+            [WebComponentsV0 ? 'createdCallback' : 'noop']() {                
+                this.createComponent();
+                // in V0, attributeChangedCallback isn't called with the initial values, we do that manually
+                Array.prototype.forEach.call(this.attributes, attr => this.attributeChangedCallback(attr.name, null, attr.value));
```

Comment:
Shouldn't this line be skipped for CEv1?

Reply:
It is, `noop` is never called.